### ISSUE TITLE
Make UI strings translatable

### ISSUE THREAD

**Issue by mynetx** [optimistdigital/nova-sortable#3]
For non-English users, the success and other messages should be translatable. How can this be achieved?

**Comment by Tarpsvo** [comment]
Hi!

I added localization support in version 1.1.0.

Just publish the translation files:

```
php artisan vendor:publish --provider="OptimistDigital\NovaSortable\ToolServiceProvider" --tag="translations"
```

And create new localizations in your project's `resources/lang/vendor/nova-sortable/` directory.

If you do translate it to some language, please PR the translation to this repository as well, thanks. :)

**Comment by mynetx** [comment]
@Tarpsvo Thank you!
I have created a new localisation in `/resources/lang/vendor/nova-sortable/de.json`, however in Nova I still see the English strings – while Nova itself is displaying from its `de.json` file. Ideas?

**Comment by Tarpsvo** [comment]
Hey!

The translation mechanism uses Laravel's core locale logic which means you have to set the app's locale to 'de' for them to be displayed. Can you confirm your app is set to 'de' locale?

**Comment by mynetx** [comment]
I can confirm that the rest of the Nova display is in German – those strings are loaded from the core German package.

**Comment by Tarpsvo** [comment]
That's weird. I tested it locally and it works just fine. Where and when are you setting the locale? I'm setting the locale in `NovaServiceProvider.php` `boot()` method, using `app()->setLocale('de')`.

**Comment by Tarpsvo** [comment]
Managed to reproduce the issue, it was indeed a bug. I overestimated the `Nova::translations()` method.

As it turned out, I have to check the `app()->getLocale()` myself in the provider code. Then find and load the correct translations file.

Reworked the translations logic and published version 1.1.1 with the fix.

**Comment by mynetx** [comment]
Thank you! 👍 I will give it a go this afternoon.
